_includes/partners/header.html

```diff
@@ -28,7 +24,7 @@
   <div class="usa-nav-container">
     <div class="usa-navbar">
       <div class="usa-logo partners-logo" id="extended-logo">
-        <a href="{{ '/partners/' | locale_url }}" title="Home" aria-label="Home">
+        <a href="{{ '/partners/' | locale_url }}" title="Home">
```

Comment:
I think this is actually a case where we might want the `aria-label`, so the link is announced as "Home" for screen readers (which is also what we do for the main site layouts). But we don't need the `title`.

```suggestion
        <a href="{{ '/partners/' | locale_url }}" aria-label="Home">
```

Reply:
Ok - updates made.